### PR TITLE
Feature/per 269

### PR DESCRIPTION
**Pull request by ChadRehmKineticData**
- An adapter was found that never got transferred to this repo
- Tests for the new adapter were set up to learn about the usage for documentation
- The readme was updated

This PR can be squashed

### CODE REVIEW

File: bridge-adapters/kinetic-bridgehub-adapter-amazonec2/pom.xml [hunk 1]
```diff
@@ -2,45 +2,57 @@
...
-    <artifactId>kinetic-bridgehub-adapter-amazonec2-v1</artifactId>
-    <version>1.0.2</version>
+    <artifactId>kinetic-bridgehub-adapter-amazonec2</artifactId>
+    <version>2.0.0</version>
     <packaging>jar</packaging>
 
-    <name>kinetic-bridgehub-adapter-amazonec2-v1</name>
+    <name>kinetic-bridgehub-adapter-amazonec2</name>
     <scm>
         <connection>scm:git:https://github.com/kineticcommunity/kinetic-bridgehub-adapter-amazonec2.git</connection>
```
Comment (jackboespflug) [inline bridge-adapters/kinetic-bridgehub-adapter-amazonec2/pom.xml:11]: Is this correct?

**Review by jackboespflug — COMMENTED** [review]
Can the `bridge-adapters/kinetic-bridgehub-adapter-amazonec2/lib/kinetic-bridgehub-adapter-1.0.3.jar` file be removed? It doesn't look like it is referenced or used anywhere.

Also, the scm configurations in the pom.xml files appear to reference the old repositories.

File: bridge-adapters/kinetic-bridgehub-adapter-amazonec2-v1/pom.xml [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.kineticdata.bridges.adapter</groupId>
+    <artifactId>kinetic-bridgehub-adapter-amazonec2-v1</artifactId>
+    <version>1.0.2</version>
+    <packaging>jar</packaging>
+
+    <name>kinetic-bridgehub-adapter-amazonec2-v1</name>
+    <scm>
+        <connection>scm:git:https://github.com/kineticcommunity/kinetic-bridgehub-adapter-amazonec2.git</connection>
```
[the remark this answers is not in the document]
Comment (ChadRehmKineticData) [reply bridge-adapters/kinetic-bridgehub-adapter-amazonec2-v1/pom.xml:11]: This is the case for all of our bridge adapters.  I'd like to address this with a bigger effort.

File: bridge-adapters/kinetic-bridgehub-adapter-amazonec2/pom.xml (hunk 1 above)
Comment (ChadRehmKineticData) [reply bridge-adapters/kinetic-bridgehub-adapter-amazonec2/pom.xml:11]: Yes this is correct.

**Pull request merged**